utils/mkrest.py

```diff
@@ -80,10 +80,8 @@ def read_file(name):
 tmp_data = read_file(tmp_file)
 if tmp_data:
     sys.stdout.write(tmp_data)
-
-process = subprocess.Popen(
-    "pandoc -s -r html %s -w rst" % src_file, shell=True, stdout=subprocess.PIPE
-)
+arguments = ["pandoc", "-s", "-r", "html", src_file, "-w", "rst"]
+process = subprocess.Popen(arguments, stdout=subprocess.PIPE)
```

Comment:
Great! Separating arguments like this is indeed a better practice. I remember reading something that said that some cases of bugs (or malfunctions) are able to be avoided by that.

Removing shell=True seems right too when reading this https://docs.python.org/3/library/subprocess.html#security-considerations and the rest of the page. 
It has been there for at least 12 years, and not the same attention was made before. 
They also suggest for maximum reliability to use fully qualified paths. But we probably don't know where it is on every known platform, so the current state seems right.